src/components/Checkout/CheckoutDropInPaymentWrapper.vue

```diff
@@ -8,6 +8,39 @@
 				:payment-types="paymentTypes"
 				@transactions-enabled="enableCheckoutButton = $event"
 			/>
+			<div v-if="isGuestCheckout" id="guest-checkout">
+				<label class="input-label" for="email">
+					Where should we email your receipt?
+					<input
+						type="email"
+						name="email"
+						v-model="email"
+						id="email"
+						class="fs-exclude"
+					>
+					<p v-if="!$v.email.email" class="input-error">
+						Valid email required.
+					</p>
+				</label>
+				<kv-checkbox
+					id="termsAgreement"
+					class="checkbox"
+				>I have read and agree to the <a
+					:href="`https://${this.$appConfig.host}/legal/terms`"
+					target="_blank"
+					title="Open Terms of Use in a new window">Terms of Use</a> and <a
+						:href="`https://${this.$appConfig.host}/legal/privacy`"
+						target="_blank"
+						title="Open Privacy Policy in a new window">Privacy Policy</a>.
+				</kv-checkbox>
+				<kv-checkbox
+					id="emailUpdates"
+					class="checkbox"
+					name="emailUpdates"
+					:checked="true"
+				>Receive email updates from Kiva (including borrower updates and promos). You can unsubscribe anytime.
+				</kv-checkbox>
```

Comment:
These checkboxes are green, but the mocks have them as blue to match with the checkbox in the Braintree dropin. I think we should discuss what we want to do about that in our team meeting and decide if we want to change that as a part of this ticket or as part of another.